### PR TITLE
[Feature] [WIP] support page

### LINKED ISSUE #221: [Feature] add support link

### LINKED ISSUE #509: [Feature] Add option to reset intros
Allow a user to reset all their tutorials on support page.
requires #221

### PR DESCRIPTION
**Pull request by petersonbrock**
closes #221, closes #509 

waiting on EEO article and any other articles that might be necessary

screen shot 2019-03-05 at 1 59 21 pm
screen shot 2019-03-05 at 1 59 31 pm
screen shot 2019-03-05 at 1 59 41 pm

### CODE REVIEW

File: app/routes/account/help/topic.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import Route from '@ember/routing/route';
+import { articles } from 'granite/config/help';
+
+export default Route.extend({
```
Comment (James1x0) [inline app/routes/account/help/topic.js]: Could you add a dynamic title token based on the name?

File: app/templates/account/help/index.hbs [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<div class="ui raised padded segment container clearfix">
+  <hgroup class="clearfix">
+    <h1 class="ui header left floated">
+      Help & Support
```
Comment (James1x0) [inline app/templates/account/help/index.hbs]:
```suggestion
      Help &amp; Support
```

File: app/templates/account/help/topic.hbs [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  {{/link-to}}
+  <hgroup class="clearfix">
+    <h1 class="ui header left floated">
+      {{model.title}}
+    </h1>
+  </hgroup>
+
+  <div class="ui divider"></div>
+  {{! template-lint-disable }}
+  {{{model.body}}}
```
Comment (James1x0) [inline app/templates/account/help/topic.hbs]: I think we can use "template-lint-disable-line" ?

File: app/config/help.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+<p>Employee counseling can be seen by navigating to an employee’s page, and then by clicking on the tab labeled “Counseling”.  To add a new issue for the employee, click the plus button in the upper righthand corner.  When creating an employee issue, specific users can be excluded from seeing the issue process by selecting their name in the dropdown for that setting.</p>
+<h3><a id="Issuing_Corrective_Action_2"></a>Issuing Corrective Action</h3>
+<p>To issue a corrective action for an employee issue, navigate to the Counseling tab for the employee and click the button labeled “Issue a New Corrective Action”, for the appropriate issue.  Creating a formal corrective action will have more options than one that is informal.  Formal corrective actions can easily be printed for presentation with the print link in the top right corner of the corrective action’s summary page.  Corrective actions can be followed up, edited, or deleted by clicking the wrench icon to the right of the corrective action’s name.</p>`
+  },
+
+  'eeo-affirmative-action': {
+    title: 'EEO & Affirmative Action',
+    body:  'James, please do this.'
```
Comment (James1x0) [inline app/config/help.js]: Haha, will do

File: app/templates/account/help/topic.hbs (hunk 3 above)
Comment (petersonbrock) [reply app/templates/account/help/topic.hbs]: not a thing that I could find...

**Pull request merged**